openapi/v2/core-metadata.yaml

```diff
@@ -57,9 +57,9 @@ components:
         onChange:
           type: boolean
           description: OnChange indicates whether the device service will generate an event only, if the reading value is different from the previous one. If true, only generate events when readings change
-        resource:
+        sourceName:
           type: string
-          description: Resource indicates the name of the resource in the device profile which describes the event to generate
+          description: SourceName indicates the name of the resource in the device profile which describes the event to generate
```

Comment:
should this be `name of the resource or device command` ?

Reply:
Modified. 
Yes, it's AutoEvent's SourceName and it should be the resource or device command.